app/v1/connectors/RetrieveUkPropertyBsasConnector.scala

```diff
@@ -28,7 +28,7 @@ import scala.concurrent.{ExecutionContext, Future}
 class RetrieveUkPropertyBsasConnector @Inject()(val http: HttpClient,
                                                 val appConfig: AppConfig) extends BaseDesConnector {
 
-  def retrievePropertyBsas(request: RetrieveUkPropertyBsasRequestData)(
+  def retrieve(request: RetrieveUkPropertyBsasRequestData)(
```

Comment:
In both calc and biss we use "retrieve___" rather than just retrieve. You've also left listBsas and triggerBsas using the old convention.

Reply:
so shall we go with retrieveBsas?